examples/MLIP_example.ipynb

```diff
@@ -422,24 +420,24 @@
     "\n",
     "dyn.attach(traj.write, interval=10)\n",
     "\n",
-    "dyn.run(2000)"
+    "dyn.run(200)"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "view(read(filename,':'))"
+    "Uncomment and run the cell below to examine the trajectory using the ASE viewer:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "#view(read(filename,':'))"
```

Comment:
Uncommenting this line here I still get this error, even after upgrading the version of ase (my current version is 3.21.1)

NameError                                 Traceback (most recent call last)
<ipython-input-24-f100c2827c1c> in <module>
----> 1 view(read(filename,':'))

NameError: name 'filename' is not defined

Reply:
Ah ok, that's not an error with ASE. Did you try running this without executing the previous cell first? It needs that cell to define the `filename` variable (that's what the error is about) and run the dynamics.